modules/phase_field/src/ics/PolycrystalVoronoiVoidIC.C

```diff
@@ -84,15 +81,6 @@ PolycrystalVoronoiVoidIC::initialSetup()
   if (_op_num <= _op_index)
     mooseError("op_index is too large in CircleGrainVoidIC");
 
-  MooseRandom::seed(getParam<unsigned int>("rand_seed"));
```

Comment:
This block of code is replicated in an intermediate base class - part of the problem with the div by zero.